src/ray/common/id.cc

```diff
@@ -155,6 +153,13 @@ const ActorHandleID ComputeNextActorHandleId(const ActorHandleID &actor_handle_i
   return ActorHandleID::FromBinary(std::string(buff, buff + ActorHandleID::Size()));
 }
 
+JobID JobID::FromInt(uint32_t value) {
+  std::vector<uint8_t> data(JobID::Size(), 0);
```

Comment:
That's a very roundabout way of doing this. I'd suggest we replace the id_ field in JobId to be of type uint32_t and then we can just directly assign. The FromBinary function can probably just be removed.

Reply:
Actually I agree that this might a bit roundabout. But I think it's hard to remove `FromBinary()` in this PR because we defined `job_id` as `bytes` in many pb files. So that we need change the type in pb files as well. I'd like to keep this PR as clean as possible.

Reply:
I think if we want to make these changes in the pb files, they should be made as part of this PR (I assume there wouldn't be that many changes). We should avoid accumulating technical debt.

Reply:
I agree that we shouldn't leave technical debt, but actually I'd like to merge this PR as quickly as possible because this blocks some other features, like `direct actor call`, `clean up GCS data`. We should develop these features parallelly. Or image this is the `API` of `JobID`,  we change the API first to unblock others and will do change the implementation later.

Reply:
Ok, then please also add a TODO for this method and we can merge the PR.

Reply:
Thanks.

Reply:
Besides the protobuf changes, other issue is that `JobID` can no longer inherit from `BaseID`. then we'll need to write a lot of duplicate code. I think it's okay to store bytes internally, but only expose `FromInt` API.